src/ORM/DataObject.php

```diff
@@ -964,7 +964,7 @@ public function classDescription(): ?string
     public function i18n_classDescription(): ?string
     {
         $notDefined = 'NOT_DEFINED';
-        $description = _t(static::class.'.CLASS_DESCRIPTION', $this->classDescription() ?? $notDefined);
+        $description = _t(static::class . '.CLASS_DESCRIPTION', $this->classDescription() ?? $notDefined);
```

Comment:
Linting issue

Reply:
I assume you mean "this fixes the linting issue"
I spent a good five minutes trying to figure out what the linting issue was with this, why CI was green, and why you were pointing it out rather than fixing it. Full sentences save time :p